pennylane/workflow/qnode.py

```diff
@@ -1176,7 +1000,7 @@ def _impl_call(self, *args, **kwargs) -> qml.typing.Result:
             override_shots = False
         else:
             if "shots" not in kwargs:
-                kwargs["shots"] = _get_device_shots(self._original_device)
+                kwargs["shots"] = self._original_device.shots

```

Comment:
```suggestion
                kwargs["shots"] = self.device.shots

```
No need to use `_original_device` anymore.